src/arrayMethodSort.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
+// prettier-ignore
 /**
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(
+    compareFunction = (a, b) => String(a) > String(b)
```

Comment:
code will be more readable if you will move defaukt fuction in the variable outside fucntion